src/controllers/comment.js

```diff
@@ -14,11 +15,32 @@ export default class CommentController {
     const { commentBody } = req.body;
     try {
       const userId = req.user.id;
+      const { lineManagerId } = req.user;
       const { requestId } = req.params;
-
       const newComment = await Comment.create({
         commentBody, requestId, userId
       });
+
+      const request = await Request.findOne({
+        where: {
+          id: requestId
+        }
+      });
+
+      const emitMessage = 'New Comment';
+
+      const notificationData = {
+        receiver: '',
+        content: emitMessage,
+        type: 'comment'
+      };
+
+      if (userId === request.lineManagerId) { notificationData.receiver = request.userId; } else { notificationData.receiver = lineManagerId; }
```

Comment:
@evansinho can we destructure the lineManageId from request, as well as the receiver from notificationData and userId from request. 
Exmaple: 
const {lineManagerId, userId} = request
const {receiver}= notificationData
if (userId === lineManagerId) { notificationData.receiver = userId; } else { receiver = lineManagerId; }

Also, since userId is also coming from req.user.id above, you can reassign it to another variable like this.
const {lineManagerId, userId: id} = request